libshaderc/src/shaderc_cpp_test.cc

```diff
@@ -76,7 +76,9 @@ class CppInterface : public testing::Test {
   // Compiles a shader and returns true on success, false on failure.
   bool CompilationSuccess(const std::string& shader,
                           shaderc_shader_kind kind) const {
-    return compiler_.CompileGlslToSpv(shader.c_str(), shader.length(), kind)
+    return compiler_
+        .CompileGlslToSpv(shader.c_str(), shader.length(), kind,
+                          "shader" /*input_file_name*/)
```

Comment:
Especially in test code, you don't need to repeat the comment about the argument.  It gets repetitive.
Coders' choice (suggest you keep it, to save work.)